### PR TITLE
[KEYCLOAK-7680] Include OIDC, SAML, and JavaScript RH-SSO client adapters

### PR DESCRIPTION
**Pull request by iankko**
(of version "4.0.0.Final-redhat-1") into RH-SSO image

Also remove the default versions of the OIDC and SAML RH-SSO client adapters, as included in the image mounting the extension directory

Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by iankko** [comment]
@rcernich @ASzc @luck3y PTAL (once got a chance)

Thank you, Jan

File: modules/sso-adapters/install [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+# Add Red Hat Single Sing-On client adapters
```
Comment (luck3y) [inline modules/sso-adapters/install]: Sign-on

**Review by luck3y — APPROVED** [review]

**Comment by iankko** [comment]
@maschmid @Crumby PTAL (once got a chance)

Thank you, Jan

**Comment by iankko** [comment]
@luck3y Thanks for that typo catch! Fixed

**Comment by luck3y** [comment]
@iankko np, thanks for teaching me what ${!foo} does :)

**Review by rcernich — APPROVED** [review]
LGTM.  You might consider changing the module name to use a package style name, e.g. keycloak.openshift.clients, or redhat.sso.openshift.clients, etc., whatever you think is best.

**Review by ASzc — COMMENTED** [review]
In my attempt to understand what the script was doing, I ended up simplifying the whole thing. I've added a commit to the PR branch with my changes.

I tried a local build with my changes and this is what is in the image:

```
[aszczucz@tyrfing ~]$ docker run --rm -ti redhat-sso-7-tech-preview/sso-cd:latest /bin/bash
[jboss@c366ad278161 ~]$ ls -lhR /opt/rh/rh-sso/client/
/opt/rh/rh-sso/client/:
total 12K
drwxr-xr-x. 2 root root 4.0K Jun 26 00:06 eap6
drwxr-xr-x. 2 root root 4.0K Jun 26 00:06 eap7
drwxr-xr-x. 2 root root 4.0K Jun 26 00:06 js

/opt/rh/rh-sso/client/eap6:
total 7.7M
-rwxr-xr--. 1 root root  350 Jun 26 00:06 install.sh
-rw-rw-r--. 1 root root 6.6M Jun 25 23:31 keycloak-eap6-adapter.zip
-rw-rw-r--. 1 root root 1.1M Jun 25 23:31 keycloak-saml-eap6-adapter.zip

/opt/rh/rh-sso/client/eap7:
total 812K
-rwxr-xr--. 1 root root  350 Jun 26 00:06 install.sh
-rw-rw-r--. 1 root root 760K Jun 25 23:31 keycloak-eap7-adapter.zip
-rw-rw-r--. 1 root root  48K Jun 25 23:31 keycloak-saml-eap7-adapter.zip

/opt/rh/rh-sso/client/js:
total 5.5M
-rwxr-xr--. 1 root root   17 Jun 26 00:06 install.sh
-rw-rw-r--. 1 root root 5.5M Jun 25 23:31 keycloak-js-adapter.zip
[jboss@c366ad278161 ~]$ head -n9999 /opt/rh/rh-sso/client/*/install.sh
==> /opt/rh/rh-sso/client/eap6/install.sh <==
#!/bin/sh
set -eu

SCRIPT_DIR="$(dirname "$0")"
# Remove the default version of the RH-SSO adapters
rm -rf /opt/eap/modules/system/add-ons/keycloak

unzip -o "$SCRIPT_DIR/keycloak-eap6-adapter.zip" -d "$JBOSS_HOME"
unzip -o "$SCRIPT_DIR/keycloak-saml-eap6-adapter.zip" -d "$JBOSS_HOME"

chown -R jboss:root "$JBOSS_HOME"
chmod -R g+rwX "$JBOSS_HOME"

==> /opt/rh/rh-sso/client/eap7/install.sh <==
#!/bin/sh
set -eu

SCRIPT_DIR="$(dirname "$0")"
# Remove the default version of the RH-SSO adapters
rm -rf /opt/eap/modules/system/add-ons/keycloak

unzip -o "$SCRIPT_DIR/keycloak-eap7-adapter.zip" -d "$JBOSS_HOME"
unzip -o "$SCRIPT_DIR/keycloak-saml-eap7-adapter.zip" -d "$JBOSS_HOME"

chown -R jboss:root "$JBOSS_HOME"
chmod -R g+rwX "$JBOSS_HOME"

==> /opt/rh/rh-sso/client/js/install.sh <==
#!/bin/sh
exit 1
```

**Comment by iankko** [comment]
@rcernich Per your suggestion renamed the ```sso-adapters``` module to more convenient / suitable ```keycloak.openshift.clients``` one.

**Comment by iankko** [comment]
@ASzc Thank you for the changes. They look great! Verified, the custom adapters are properly expanded and installed with them too. Unless someone else has objections, I am OK with the current form of this PR.

**Review by ASzc — APPROVED** [review]

**Comment by ASzc** [comment]
Will merge and build in a few hours, unless any objections are raised before then.

**Comment by iankko** [comment]
@ASzc Thanks, Alex!

**Pull request merged**